docs/reference/modules/network.asciidoc

```diff
@@ -24,7 +24,7 @@ host to other nodes in the cluster. Accepts an IP address, hostname, a
 <<network-interface-values,special value>>, or an array of any combination of
 these. Note that any values containing a `:` (e.g., an IPv6 address or
 containing one of the <<network-interface-values,special value>>) must be quoted
-because `:` is a special character in YAML.
+because `:` is a special character in YAML. `0.0.0.0` is an acceptable IP address, it has the same effect as the special value `0`.
```

Comment:
I think this is fine. As I still would not document `0` as "special value" (see rationale below), I think you could merge the text to say something along the lines of:

> `0.0.0.0` is an acceptable IP address and will binds to all network interfaces.

Can you please also wrap text after 80 characters in docs?

Reply:
My primary motivation for this PR is to document the value `0`, as I found that in some config examples and had hard time to find what that means.

Do you agree with:
> `0.0.0.0` is an acceptable IP address and will bind to all network interfaces. Value `0` has the same effect as value `0.0.0.0`.

?

Reply:
Yes, I am ok with this change. Just a very minor nit; the articles are missing, i.e.

> The value `0` has the same effect as the value `0.0.0.0`.

Reply:
@GoodMirek I think we're very close to completion now. Are you still interested in getting your change in? :)

Reply:
@danielmitterdorfer Will look next week. What is a process for merging these changes to current master or whatever the development branch is?
Will I have to go through the whole process again?

Reply:
Thanks @GoodMirek. There is no need to concern yourself with other branches than this one. I will take care of merging this to the correct branches on your behalf via cherry-picking (so you will still show up as the author of the change).

Reply:
@GoodMirek Any chance you can push the commit including our final changes? Then I can take it from there.